script/release/release.js

```diff
@@ -5,20 +5,16 @@ if (!process.env.CI) require('dotenv-safe').load();
 const args = require('minimist')(process.argv.slice(2), {
   boolean: [
     'validateRelease',
-    'skipVersionCheck',
```

Comment:
is this just unused? we also pass this to Sudowoodo - we should delete if thats the case